### PR TITLE
[OSDOCS-10334]:ROSA with HCP Upgrade Section

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.15+
Issue:

https://issues.redhat.com/browse/OSDOCS-10334
Link to docs preview:

https://75366--ocpdocs-pr.netlify.app/openshift-rosa/latest/upgrading/rosa-hcp-upgrading.html
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @xueli181114, could I please get a review for this PR when you get a moment. Thanks!

**Review by radtriste — APPROVED** [review]
/lgtm

File: upgrading/rosa-hcp-upgrading.adoc [hunk 1]
```diff
@@ -12,6 +12,7 @@ Use one of the following methods to upgrade your HCP clusters:
 
 * Upgrade only your hosted control plane. This does not impact your worker nodes.
 * Upgrade only your machine pool. This initiates a rolling reboot of a specific machine pool and temporarily impacts the worker nodes on the specific machine pool. It does not impact all your worker nodes if you have multiple machine pools.
+* Upgrade multiple machine pools simultaneously. This will initiate a rolling reboot of just updated machine pools. This allows for updating multiple nodes simultaneously within a cluster.
```
Comment (mburke5678) [inline upgrading/rosa-hcp-upgrading.adoc:15]: > just updated machine pools

I'm not sure of what this means. 
Only updated machine pools?
Recently updated machine pools?

Comment (mletalie) [reply upgrading/rosa-hcp-upgrading.adoc:15]: Hello @mburke5678. Good question. I asked PM and he agreed it needed to be clarified. Will make changes based on the info he provided.  Thx.

**Pull request merged**